contracts/SpokePool.sol

```diff
@@ -883,12 +959,40 @@ abstract contract SpokePool is
         }
     }
 
+    function _updateCountFromFill(
+        uint256 startingFillAmount,
+        uint256 endingFillAmount,
+        uint256 totalFillAmount,
+        int64 realizedLPFeePct,
+        address token,
+        bool localRepayment,
+        bool useContractFunds
+    ) internal {
+        // If this is a slow fill or it's an initial 0-fill, do nothing, as these should not impact the count.
+        if (useContractFunds || endingFillAmount == 0) return;
```

Comment:
I don't think we allow 0 fills, only 1 wei fills

Reply:
But it is possible to send 0 fills, right?

Reply:
it is yes